src/main/java/uk/gov/hmcts/reform/divorce/orchestration/framework/workflow/RetryableBulkCaseWorkflow.java

```diff
@@ -96,14 +130,18 @@ public boolean executeWithRetries(Map<String, Object> bulkCaseResponse, String b
                 caseId = String.valueOf(caseLink.get(CASE_REFERENCE_FIELD));
                 this.run(bulkCaseData, caseId, authToken);
             } catch (FeignException.BadGateway
-                | FeignException.InternalServerError
-                | FeignException.GatewayTimeout
-                | FeignException.ServiceUnavailable
-                | RetryableException e) {
+                    | FeignException.InternalServerError
+                    | FeignException.GatewayTimeout
+                    | FeignException.ServiceUnavailable
+                    | RetryableException e) {
                 String errorMessage = e.content() == null ? e.getMessage() : e.contentUTF8();
                 log.error("Case update failed, added to retry list: for bulk case id {} and caseId {}. Cause {}",
                     bulkCaseId, caseId, errorMessage, e);
                 casesToRetry.add(caseElem);
+            } catch (FeignException.UnprocessableEntity e) {
+                log.error("Case update failed with 422 error : for bulk case id {}  and caseId {}", bulkCaseId, caseId, e);
```

Comment:
can you add this on the error message to show properly the error? 
```suggestion
                String errorMessage = e.content() == null ? e.getMessage() : e.contentUTF8();
                log.error("Case update failed with 422 error : for bulk case id {}  and caseId {} : errorMessage", bulkCaseId, caseId, e);
```

Reply:
I'm also thinking if we can get 404?

Reply:
Do we want to drop 404 cases or does that mean something wrong with the bulk case? I can add it in if so.